en/Obsidian/Community code of conduct.md

```diff
@@ -52,9 +68,9 @@ Please flag the relevant conversation with a reason, and our moderator team will
 > A verified major offense results in an immediate ban.
 > A verified minor offense results in a warning; after that a second verified violation results in a ban.
 
-If our moderator team witnesses or receives a report about a [[#major offense]], we will contact the offender to have a conversation with them and verify what has happened, and they will be removed from our community.
+If our moderator team witnesses or receives a report about [[#serious offenses]], we will contact the offender to have a conversation with them and verify what has happened, and they will be removed from our community.
```

Comment:
> [...] have a conversation with them and verify what has happened, and they will be removed from our community

This potentially sounds like they will be banned regardless of their defense?

Reply:
I'll change it too "we will contact the offender to have a conversation with them and verify what has happened. If verified, they will be removed from our community".

Major offenses are only harassments and threats. I don't think there's much room for defense as they are pretty clear cut.